fe/fe-core/src/main/java/com/starrocks/sql/optimizer/MvRewritePreprocessor.java

```diff
@@ -251,13 +250,6 @@ private LogicalOlapScanOperator createScanMvOperator(MaterializationContext mvCo
                 mvConjuncts.add(conj);
```

Comment:
Do we still need `mvConjuncts ` anymore if no partition prunes?

Reply:
remove it here, move it to rewriter.